Task 3/system/Coding/src/app/PositionControl.h

```diff
@@ -25,30 +25,26 @@
 
 /* PROTOTYPES *************************************************************************************/
 /** 
- * @brief starts timer1 to measure the Time in SearchTrackState
+ * @brief updates the sensorvalues, must be called before any of the other functions are beein called
```

Comment:
See previous comments regarding imperative form of doxgen documentation. Won't note this for other functions anymore.

```suggestion
 * @brief Update the sensorvalues, must be called before any of the other functions are being called.
```